packages/api/helpers.ts

```diff
@@ -0,0 +1,8 @@
+import { ErrorResponse } from '@translation/api-types';
+import { NextApiRequest, NextApiResponse } from 'next';
+
+export type ApiRequest<Params = void> = Params extends void
+  ? NextApiRequest
+  : NextApiRequest & { query: Params };
+
+export type ApiResponse<Body = void> = NextApiResponse<Body | ErrorResponse>;
```

Comment:
These helpers will make it easier to type the router parameters, and to allow the response json to include error responses along side with the 200 response.